docs/RELEASE.md

```diff
@@ -27,3 +27,9 @@ SPDX-License-Identifier: LGPL-2.1-or-later
 18. [FINAL] Build and upload the documentation (on the -stable branch): `ninja -C build doc-sync`
 20. [FINAL] Change the Github Pages branch to the newly created branch (https://github.com/systemd/systemd/settings/pages) and set the 'Custom domain' to 'systemd.io'
 21. [FINAL] Update version number in `meson.version` to the devel version of the next release (e.g. from `v256` to `v257~devel`)
+
+# Steps to a Successful Stable Release
+
+1. Backport the commits from all PRs tagged with `needs-stable-backport` on Github with `git cherry-pick -x`. Since 256 the stable branches live [here](https://github.com/systemd/systemd/). Stable branches for older releases are available [here](https://github.com/systemd/systemd-stable/). Check each commit to see if it makes sense to backport and check the comments on the PR to see if the author indicated that only specific commits should be backported.
```

Comment:
This should be "at least the commits from..." as we pick more, as it makes sense, for commits that fix bugs, change docs/man/tests/ci